packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -54,6 +54,8 @@ export const messages = {
   parameter_gatherer_enter_username_name: 'Enter target username',
   parameter_gatherer_enter_alias_name:
     'Enter an org alias or use default alias',
+  parameter_gatherer_enter_custom_url:
+    'Enter a custom login URL or use default URL',
```

Comment:
I assume the default URL will show in the text input box?

Reply:
Right, it shows as the placeholder in the input box similar to the org alias.